app/screens/post_options/post_options.tsx

```diff
@@ -69,7 +73,7 @@ const PostOptions = ({
 
     const renderContent = () => {
         return (
-            <>
+            <ScrollView>
```

Comment:
Should have scrolling disabled if jo bindings are present

Reply:
Maybe I missed it, but I'm not seeing where the snap point of the bottom sheet is added when there are bindings so that it (bottom sheet) can be expanded up to 90% of the screen height, well I'm not sure if that is part of the design or not, but in my mind it made sense

Reply:
> Maybe I missed it, but I'm not seeing where the snap point of the bottom sheet is added when there are bindings so that it (bottom sheet) can be expanded up to 90% of the screen height, well I'm not sure if that is part of the design or not, but in my mind it made sense

@enahum I'm not familiar with how to do this. Can you point me to a similar solution in another scenario?

Reply:
@mickmister basically here https://github.com/mattermost/mattermost-mobile/pull/6735/files#diff-54601045814137800ace02ee7d0134d8cfb24c938ac979c85bde85ae9399b0faR149 you have the snap points of the bottom sheet (which is an array representing the point in the Y axis, consider that as the height of the bottom sheet).

Snap points are set from top to bottom, for the post options you'll see that is the height of each option multiplied by the amount of options, and the second value in the array is the value `10` meaning that the second snap point is a height of 10 from the bottom of the screen, which will cause the bottom sheet to close. Another type of value that the bottom sheet accepts as a snap point is a percentage, like `'90%'` (notice percentages are strings)

Also as you can see as one of the props there is `initialSnapIndex` and this one is hardcoded at the moment to 0, which is the position of the value in the snapPoints array, so here we are saying when showing, start at the position (amount of option items * height of option items)

What I expect is that, when bindings are set, the snapPoints array contains the values `['90%', ((snapPoints + additionalSnapPoints) * ITEM_HEIGHT), 10]` and the initialSnapIndex to be set to 1 so that the bottom sheet continues to open as is today but that can be dragged almost all the way to the top of the screen when there are bindings set.

Reply:
@enahum For the purpose of 1.0, I think the current behavior of the PR is fine for the current state of Apps. It mainly needs to be functional as we continue to develop the Apps. We can essentially assume that there will be no Apps installed at the moment.

However if there is something wrong here and these changes you suggest are required for this PR, is someone from your team able to implement this? I'm not very familiar with RN or the details of what you're suggesting

Reply:
@enahum @larkox What are your thoughts on deferring this item? @javaguirre Do you think you would be able to help implement this?

Reply:
I am ok deferring this item to the next release based in the assumption we will be having 2 or 3 post menu bindings for now.

Reply:
@enahum I changed the code to make room for at most 4 bindings,. I tried the `90%` snap point, but that ends up with a lot of extra whitespace at the bottom of the sheet:

<img width="539" alt="image" src="https://user-images.githubusercontent.com/6913320/202577949-07cfd1ee-df79-40f3-841f-42763210ffc6.png">

<img width="602" alt="image" src="https://user-images.githubusercontent.com/6913320/202578132-eae2131f-eee6-4015-99ae-6d75ec307cd3.png">

I'm thinking we should get a UX opinion on this. Who would be the go-to person to discuss this with?

Reply:
For UX that would be @matthewbirtch 

But the approach about modifying the snap point like this `snapPoints += Math.min(bindings.length, 4);` is not what I had in mind, but could work if instead of going full height, we just give it enough so that the user understanda that they can scroll for more items. Makes sense?

Reply:
@enahum Here are two different options:

Partially showing the first binding. I'm thinking this one is best, since the binding options are not going to be as used as the other options.

<img width="300" alt="image" src="https://user-images.githubusercontent.com/6913320/203179894-2d8a3d5d-a228-4b6c-bea7-66e597389441.png">

Showing the first binding in whole, and partially showing the second binding

<img width="300" alt="image" src="https://user-images.githubusercontent.com/6913320/203179996-9fb1f807-6f7a-49eb-81e5-bdc8676666f5.png">


-----

In either case, here's what it looks like expanded:

<img width="300" alt="image" src="https://user-images.githubusercontent.com/6913320/203179980-a5761cf3-dc21-4e3c-be27-80d15b200025.png">


Here's what it looks like with 8 options:

<img width="300" alt="image" src="https://user-images.githubusercontent.com/6913320/203180342-f92835a0-2c16-4858-9c69-363116bb6316.png">

14 options:

<img width="300" alt="image" src="https://user-images.githubusercontent.com/6913320/203180550-928c2a7a-7fc8-4567-850a-3ac4e275df9e.png">